includes/TinyMCE.php

```diff
@@ -108,14 +109,32 @@ public function web_stories_mce_plugin( array $plugins ) {
 	public function register_assets() {
 		$this->enqueue_style( 'wp-components' );
 
-		// Can't use Assets::enqueue_script() because the script needs to be loaded via the mce_external_plugins filter.
-		$asset = $this->get_asset_metadata( 'tinymce-button' );
+		$this->enqueue_script( self::SCRIPT_HANDLE );
+		wp_localize_script(
+			self::SCRIPT_HANDLE,
+			'webStoriesData',
+			$this->get_script_data()
+		);
+	}
 
-		// Only works when wp_register_script and wp_enqueue_script are called separately.
-		wp_register_script( self::SCRIPT_HANDLE, false, $asset['dependencies'], $asset['version'], true );
-		wp_add_inline_script( self::SCRIPT_HANDLE, 'var webStoriesData = ' . wp_json_encode( $this->get_script_data() ) . ';' );
+	/**
+	 * High jack the tinymce to render an empty script tag for tinymce.
+	 *
+	 * @since 1.5.0
+	 *
+	 * @param string $tag    The `<script>` tag for the enqueued script.
+	 * @param string $handle The script's registered handle.
+	 * @param string $src    The script's source URL.
+	 *
+	 * @return string $tag The `<script>` tag for the enqueued script.
+	 */
+	public function script_loader_tag( $tag, $handle, $src ) {
+		if ( self::SCRIPT_HANDLE === $handle ) {
+			$tag = str_replace( $src, '', $tag );
+			$tag = (string) preg_replace( '#<script src=\'\'(.*?)>(.*?)</script>#is', '', $tag );
```

Comment:
```suggestion
			return '';
```

does this also work?

Reply:
Nope. This would result in all extras not printing out. 

https://github.com/WordPress/wordpress-develop/blob/1479f1e5438846ca655634361f0fb4553e7e9436/src/wp-includes/class.wp-scripts.php#L387-L400